### DISCUSSION TITLE
[SOLVED] Docmost does not show content if I use a domain or behind a reverse-proxy

### DISCUSSION THREAD

**Discussion by alohl669** [docmost/docmost#219] [Help / Questions] [answered]
Hi, I have a private domain and I can't see any content on docmost when I use it.
![image](https://github.com/user-attachments/assets/2b8779cd-56ca-4501-a8b5-814d3fc62694)

I have my docmost service behind nginx.
nginx and docmost are in different virtual machines

I'm using `image: docmost/docmost:latest` almost as you describe [here](https://docmost.com/docs/installation)

> I use the right `APP_URL` var for my case and I added `container_name` fields for better control

**Comment by Philipinho** [comment]
Can you please check the browser console for any errors?

**Comment by alohl669** [comment]
![image](https://github.com/user-attachments/assets/2faf4f1e-fce1-4852-acc6-d1a36f7ea52f)

**Reply by Philipinho** [reply, marked as answer]
Is your NGINX config setup to upgrade websocket connections?

**Reply by alohl669** [reply]
No, I didn't but it works!
Thank you!
![image](https://github.com/user-attachments/assets/bd203359-51d1-446b-b11d-f231fe8f9156)

**Comment by alohl669** [comment]
Let me make it more understandable
![docmost_issue drawio](https://github.com/user-attachments/assets/eb41290a-4d55-4ee0-8b33-9bb372f1b6c6)
